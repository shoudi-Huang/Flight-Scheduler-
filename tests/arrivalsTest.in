arrivals
arrivals Chicago
location import locations.csv
ARRIVALS Chicago
arrivals qp
flight import flights.csv
arRivALs cHiCAgo
flight 119 remove
arrivals chicago extra parameter
exit
