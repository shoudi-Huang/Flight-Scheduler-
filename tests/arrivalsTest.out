User: This location does not exist in the system.

User: This location does not exist in the system.

User: Imported 51 locations.

User: Chicago
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------

User: This location does not exist in the system.

User: Imported 169 flights.
13 lines were invalid.

User: Chicago
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
 119 Mon 11:36   Arrival from London
 124 Tue 15:36   Arrival from NewYork
  68 Wed 06:36   Arrival from NewYork
 144 Wed 08:46   Arrival from Dallas
 121 Wed 21:51   Arrival from London
 120 Thu 07:51   Arrival from London
 118 Thu 11:36   Arrival from London

User: Removed Flight 119, Mon 02:45 London --> Chicago, from the flight schedule.

User: Chicago
-------------------------------------------------------
ID   Time        Departure/Arrival to/from Location
-------------------------------------------------------
 124 Tue 15:36   Arrival from NewYork
  68 Wed 06:36   Arrival from NewYork
 144 Wed 08:46   Arrival from Dallas
 121 Wed 21:51   Arrival from London
 120 Thu 07:51   Arrival from London
 118 Thu 11:36   Arrival from London

User: Application closed.
